src/components/tabbar/item.js

```diff
@@ -0,0 +1,196 @@
+// @flow
+
+import React from "react";
+
+import {StyleSheet} from "aphrodite";
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+type Props = {
+    onPress: () => void,
+    itemState: ItemState,
+    itemType: TabbarItemType,
+};
+
+export type ItemState = "active" | "inactive" | "disabled";
+type State = {};
+
+const styles = StyleSheet.create({
+    base: {
+        display: "flex",
+        width: 44,
+        height: 38,
+        boxSizing: "border-box",
+        borderRadius: 3,
+        border: `1px solid transparent`,
+        marginRight: 1,
+        marginLeft: 1,
+    },
+    hovered: {
+        background:
+            "linear-gradient(0deg, rgba(24, 101, 242, 0.32), rgba(24, 101, 242, 0.32)), #FFFFFF",
```

Comment:
nit: the `#FFFFFF` could be `${Color.white}` similar to the `innerBoxPressed` down below.

Reply:
Done!